ericberringer/Currency-Kidz
# build: updated the currency object in deposit views create method,now…

… saves the amount for each coin/bill.

Description Fixes # (issue)

Type of change:

Currency dict in views/deposit_events create method stores individual currency counts now.

Testing Instructions

Created Deposit event object should list amount of individual coins/bills now.
/deposit_events in Postman, newest deposit should have a currency list on it.

git fetch --all 
git checkout CK-DepositCurrency

Checklist:

X(I have preformed a self review)

X(Code follows style guidelines of project)

(I have commented my code. Especially w/ hard to understand code)

X(Changes I have made generate no new warnings or errors)

X(I have added test instructions that prove my fix is effective or that my feature works)

(I am not ruminating about the things my therapist told me to forget about b/c I cannot do anything about them. For example, the state of the world. )

- Fixes #
